lib/node_modules/@stdlib/regexp/to-json/lib/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Return a JSON representation of a regular expression.
+*
+* @module @stdlib/regexp/to-json
+*
+* @example
+* var toJSON = require( '@stdlib/regexp/to-json' );
+*
+* var pattern = 'ab+c'
+* var regexp = new RegExp( pattern );
+* var json = toJSON( regexp );
```

Comment:
```suggestion
* var json = toJSON( /ab+c/ );
```

No need to use `RegExp` constructor here.